### DISCUSSION TITLE
DEF Tank / 2nd Fuel Tank

### DISCUSSION THREAD

**Discussion by redbaron73** [hargata/lubelog#699] [Ideas]
I would like to see the ability to track secondary fuel type. This could be Propane, DEF, Refer, etc.

My specific use case is a Diesel pusher that has a primary fuel tank (diesel) and a DEF tank. IN the case of DEF, MPG does make sense. Others may use this for propane to track the hours of usage, separate from the engine.

**Comment by marc115uk** [comment]
This would be handy for DEF would like to be able to see my mpg for DEF as well as my diesel
